README.md

```diff
@@ -1,6 +1,6 @@
 ![ci](https://github.com/bndw/wifi-card/workflows/ci/badge.svg)
 
-# 📶 WiFi Card
+# <img width="32px" src="/src/images/wifi.png">  WiFi Card
```

Comment:
The icon isn't showing up in the rendered diff view, but that could be a Github thing or the filepath might need to be prefixed with a dot, e.g. `./src/...`